### PR TITLE
Replace react-select

### PR DESCRIPTION
**Pull request by tiiavalt**
In this pull request `react-select` is replaced with [`react-a11y-select`](https://github.com/mdesjardins/react-a11y-select), because it's more accessible.

### CODE REVIEW

File: frontend/src/components/buttons/DropdownSelect.styles.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import caretDown from '../../images/caret-down.svg';
+
+export const DropdownWrapper = styled.div`
+    .ReactA11ySelect {
```
Comment (koops9) [inline frontend/src/components/buttons/DropdownSelect.styles.js:5]: One difference I realized that the font didn't match anymore so consider adding around here
`    font-family: 'Hack';`

**Comment by koops9** [comment]
image
Seems like the component has not been updated in 3 years and has not been updated to the newest react. Epi will probably be used in newer React so not sure what will come out of this then....

**Comment by tiiavalt** [comment]
> image
> Seems like the component has not been updated in 3 years and has not been updated to the newest react. Epi will probably be used in newer React so not sure what will come out of this then....

I didn't notice this at all. I guess we could fork the repository if this becomes an issue..? 
All in all this component was in my opinion the best choice and the other options that I looked into, even though newer, were somehow inadequate in terms of accessibility.

**Comment by vade5g** [comment]
[Replace react-select](https://trello.com/c/3sd3hpaD/199-replace-react-select)

**Review by koops9 — APPROVED** [review]
Seems at least better solution compared to last one. We'll probably have to check this if this upgrades to next React version but it will have to be dealt then.

**Pull request merged**